src/smc-webapp/jupyter/top-buttonbar.tsx

```diff
@@ -4,10 +4,15 @@ The static buttonbar at the top.
 
 import { React, Component, rclass, rtypes, Rendered } from "../app-framework";
 import * as immutable from "immutable";
-import { Button, ButtonGroup, Form, FormControl } from "react-bootstrap";
-const { Icon, VisibleMDLG, VisibleLG } = require("../r_misc");
-import { endswith } from "smc-util/misc2";
-
+import { Button, ButtonGroup, Form } from "react-bootstrap";
+const {
+  Icon,
+  VisibleMDLG,
+  VisibleLG,
+  DropdownMenu,
+  MenuItem
+} = require("../r_misc");
```

Comment:
`import` is available with `r_misc` now.

Reply:
yes. I overlooked that. I could go through the code and change many of them in bulk.